### PR TITLE
Added dummy test

### PR DESCRIPTION
**Pull request by Daniel-olaO**

### CODE REVIEW

File: stripe2qbo/api/tests/test_app.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    response = client.get("/settings")
+    assert response.status_code == 401
+
+def test_save_settings():
+    response = client.post("/settings")
+    assert response.status_code == 401
+
+def test_sync_single_transaction():
+    response = client.post("/sync")
+    assert response.status_code == 401
```
Comment (Russell-Pollari) [inline stripe2qbo/api/tests/test_app.py:32]: I suppose the plan is to update these for the happy path (status_code ==200)?

But should be explicit with the test names. Most of these are testing that the API returns an error code when a user is not logged in. Should update the test names to reflect that 

(e.g. `test_get_settings_unauthorized`)

File: stripe2qbo/db/models.py [hunk 2]
```diff
@@ -40,3 +41,14 @@ class QBOToken(Base):
     user_id: Mapped[int] = mapped_column(ForeignKey("users.id"))
 
     user = relationship("User", back_populates="qbo_token")
+
+
+class TransactionSync(Base):
```
Comment (Russell-Pollari) [inline stripe2qbo/db/models.py:46]: Looks good, but should be in a different PR. Ideally, each PR does one thing, which can be understood from the description.

In this case, I read the title "Added dummy test", and this is completely unrelated

**Review by Russell-Pollari — CHANGES_REQUESTED** [review]

File: stripe2qbo/db/models.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+from typing import Literal, Optional
```
Comment (Russell-Pollari) [inline stripe2qbo/db/models.py:1]: These I think are unused?

Did you do this locally? I'm surprised the pre-commit hook didn't catch this

File: stripe2qbo/api/tests/test_app.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+client = TestClient(app)
+
+
+def test_index():
+    response = client.get("/")
+    assert response.status_code == 200
+    assert "text/html" in response.headers["content-type"]
+
+def test_user_id_unauthorized():
```
Comment (Russell-Pollari) [inline stripe2qbo/api/tests/test_app.py:13]: There should be 2 lines btw each function. Can you make sure `pre-commit` is running? It shouldn't let you commit until the formatters/linters are appeased

https://pre-commit.com/

File: stripe2qbo/db/models.py (hunk 3 above)
Comment (Russell-Pollari) [reply stripe2qbo/db/models.py:1]: You might need to run `$ pre-commit install`

**Review by Russell-Pollari — CHANGES_REQUESTED** [review]

File: stripe2qbo/api/tests/test_app.py (hunk 4 above)
Comment (Daniel-olaO) [reply stripe2qbo/api/tests/test_app.py:13]: Added pre-commit

**Comment by Russell-Pollari** [comment]
Ok almost done. Can you move the test file to `stripe2qbo/tests/` I'd like to keep all the tests in one folder

**Review by Russell-Pollari — APPROVED** [review]

**Comment by Russell-Pollari** [comment]
thanks @Daniel-olaO !
